pkg/kubeadm/master/manifests.go

```diff
@@ -187,10 +237,9 @@ func componentPod(container api.Container, volumes ...api.Volume) api.Pod {
 }
 
 func getComponentCommand(component string, s *kubeadmapi.KubeadmConfig) (command []string) {
-	baseFalgs := map[string][]string{
+	baseFlags := map[string][]string{
```

Comment:
Yeah, the current way of doing things is making us all overlap in this kind of _fixes_.